data-processing/src/dfm_data/datatrove_pipeline/url_filter.py

```diff
@@ -0,0 +1,437 @@
+import itertools
+import re
+import os
+import smart_open
+import socket
+from typing import Generator, Optional
+
+from huggingface_hub import cached_assets_path
+from loguru import logger
+
+from datatrove.data import Document
+from datatrove.io import safely_create_file
+
+from datatrove.pipeline.writers.disk_base import DiskWriter
+from datatrove.pipeline.filters.base_filter import BaseFilter
+
+SOFT_DOMAIN_WORD_BANLIST_EXTRA = (
```

Comment:
is it used anywhere?

Reply:
It's meant for being imported into a script and be used as keyword argument `soft_banned_words` of the datatrove URL filter, which will add these words to the standard list.